From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 9 Aug 2025 19:39:52 +0200
Subject: searchd: Honor meson option
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Just having an option isn't enough, we also need
to take it into account.

Fixes: 4c58fc4fb ("searchd: Add deamon to gather search results")
Signed-off-by: Guido Günther <agx@sigxcpu.org>
Part-of: <https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1755>
---
 searchd/meson.build | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/searchd/meson.build b/searchd/meson.build
index 9681ab6..d4ee982 100644
--- a/searchd/meson.build
+++ b/searchd/meson.build
@@ -1,3 +1,7 @@
+if not get_option('searchd')
+  subdir_done()
+endif
+
 searchd_args = ['-DG_LOG_USE_STRUCTURED', '-DG_LOG_DOMAIN="mobi.phosh.Shell.Search"']
 
 executable(
